core-plugins/src/test/java/co/cask/hydrator/plugin/realtime/sink/RealtimeKinesisStreamSinkTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.realtime.sink;
+
+import co.cask.cdap.etl.api.realtime.RealtimeSink;
+import co.cask.cdap.etl.api.realtime.RealtimeSource;
+import co.cask.cdap.etl.proto.v2.ETLPlugin;
+import co.cask.cdap.etl.proto.v2.ETLRealtimeConfig;
+import co.cask.cdap.etl.proto.v2.ETLStage;
+import co.cask.cdap.proto.Id;
+import co.cask.cdap.proto.artifact.AppRequest;
+import co.cask.hydrator.common.Constants;
+import co.cask.hydrator.plugin.common.Properties;
+import co.cask.hydrator.plugin.realtime.ETLRealtimeTestBase;
+import co.cask.hydrator.plugin.realtime.source.DataGeneratorSource;
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test default configuration
+ */
+public class RealtimeKinesisStreamSinkTest extends ETLRealtimeTestBase {
+
+
+  @Test(expected = IllegalStateException.class)
+  public void kinesistest() throws Exception {
```

Comment:
kinesis is configured using api access keys, I have a unit test for checking positive test cases that i can not check in because of the presence of api keys in it.
tested on cluster